src/components/form/Input/index.js

```diff
@@ -3,15 +3,34 @@ import cn from 'classnames';
 
 import './styles.scss';
 
-const Input = ({ register, type = 'text', name, error, handleFocus, placeholder = '' }) => (
+const Input = ({
+  register,
+  type = 'text',
+  name,
+  error,
+  handleFocus,
+  placeholder = '',
+  defaultValue = null,
+}) => (
   <div className="d-flex flex-column text-center Input">
-    <input
-      className={cn({ error })}
-      type={type}
-      {...register(name)}
-      onFocus={handleFocus}
-      placeholder={placeholder}
-    />
+    {defaultValue ? (
```

Comment:
you can avoid repeating the whole component by using an optional param like so:
value={defaultValue ? defaultValue : undefined)
https://stackoverflow.com/questions/32232659/react-inline-conditionally-pass-prop-to-component